rs/nns/governance/src/governance.rs

```diff
@@ -2895,6 +2895,10 @@ impl Governance {
     /// The source neuron's stake, maturity and age are moved into the target.
     /// Any fees the source neuron are burned before the transfer occurs.
     ///
+    /// TODO: this is not true if the stake is below the transaction fee
+    /// TODO: not clear what "current fees" are below
```

Comment:
Not sure what the question here is, but I try to answer...

Neuron has a fee field. This gets increased when a neuron makes a proposal (the neuron needs to have enough ICP to cover this). If the proposal passes, it gets decreased by the same amount. At some point (disburse?), ICP gets burned to cover the fee (alternatively, I think it's possible to cover fee using staked maturity).

Reply:
Merge neuron already burns the fees of the source neuron (if these are bigger than the `TRANSACTION_FEE`). If "current fees" means the `neuron_fees_e8s` field, then the comment below:
```
 /// Current fees are not affected in either neuron.
 ```
 Is not correct for the source neuron.